pkg/kubectl/history.go

```diff
@@ -103,30 +101,24 @@ func PrintRolloutHistory(historyInfo HistoryInfo, resource, name string) (string
 		return fmt.Sprintf("No rollout history found in %s %q", resource, name), nil
 	}
 	// Sort the revisionToChangeCause map by revision
-	var revisions []string
-	for k := range historyInfo.RevisionToTemplate {
-		revisions = append(revisions, strconv.FormatInt(k, 10))
+	var revisions []int64
+	for r := range historyInfo.RevisionToTemplate {
+		revisions = append(revisions, r)
 	}
-	sort.Strings(revisions)
+	sliceutil.Ints64(revisions)
```

Comment:
i am not one of the official reviewers, but seems like we are sorting revisions inside Ints64(revisions) which seems to be hiding the intent. Why cant we just say sliceutil.sort or create such a function if it doesnt exist, instead of saying sliceutil.Ints64(revisions) and sorting underneath. Let me know if i am missing something